recipes/mitmproxy/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "mitmproxy" %}
+{% set version = "8.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mitmproxy/mitmproxy/archive/v{{ version }}.tar.gz
+  sha256: 212e81bec40e1d2a894d73a337f076a3010d981249e513565e9246f29822d06c
+
+build:
+  entry_points:
+    - mitmproxy = mitmproxy.tools.main:mitmproxy
+    - mitmdump = mitmproxy.tools.main:mitmdump
+    - mitmweb = mitmproxy.tools.main:mitmweb
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [py<39]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - asgiref >=3.2.10,<3.6
+    - blinker >=1.4,<1.5
+    - brotli-python >=1.0,<1.1
+    - certifi >=2019.9.11
+    - cryptography >=36,<38
+    - flask >=1.1.1,<2.2
+    - h11 >=0.11,<0.14
+    - h2 >=4.1,<5
+    - hyperframe >=6.0,<7
+    - kaitaistruct >=0.7,<0.10
+    - ldap3 >=2.8,<2.10
+    - msgpack-python >=1.0.0,<1.1.0
+    - passlib >=1.6.5,<1.8
+    - protobuf >=3.14,<5
+    - publicsuffix2 >=2.20190812,<3
+    - pydivert >=2.0.3,<2.2          # [win]
+    - pyopenssl >=21.0,<22.1
+    - pyparsing >=2.4.2,<3.1
+    - pyperclip >=1.6.0,<1.9
+    - python
```

Comment:
```suggestion
    - python >=3.9
```

Reply:
See https://github.com/mitmproxy/mitmproxy/blob/main/setup.py#L71

Reply:
I think that needs to be like that. Look at the linter messages in this PR:
1. Because this recipe has a windows only dependency it can't be `noarch: python`, as noarch packages have to be the same on all architectures: https://github.com/conda-forge/staged-recipes/pull/19813#issuecomment-1195692656
2.Non `noarch` packages can't specify python version constraints: https://github.com/conda-forge/staged-recipes/pull/19813#issuecomment-1195703937

Because of that I used the solution I found here: https://groups.google.com/g/conda-forge/c/vZj02SDKcO0 and added the python version as a selector to the skip instead.
This also seems to be what other Non-`noarch` python packages do. For example numpy: https://github.com/conda-forge/numpy-feedstock/blob/main/recipe/meta.yaml#L23

Reply:
You are right, sorry for the confusing suggestion!